src/core_ocean/Registry.xml

```diff
@@ -2031,6 +2031,7 @@
 	<var_struct name="state" time_levs="2">
 		<var name="normalVelocity" type="real" dimensions="nVertLevels nEdges Time" units="m s^{-1}"
 			 description="horizonal velocity, normal component to an edge"
+       missing_value="FILLVAL"
```

Comment:
@wenshanw, this spacing needs to be done with tabs rather than spaces to be compatible with the rest of the XML in MPAS.  (I know it's hard to see the difference in a lot of editors.)